### PR TITLE
Update loading screen content

### PR DESCRIPTION
**Pull request by jon-shipley**
Update SPA copy based on UX review

### CODE REVIEW

File: pupil-spa/src/app/loading/loading.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="preload" id="js-preload-div">
   <div>
     <p id="js-loading" class="font-large text-center">
-      Question {{question}} out of {{total}}
+      Question {{question}}&hellip;
```
Comment (GeorgeChatzigiannis) [inline pupil-spa/src/app/loading/loading.component.html]: @jon-shipley what's &hellip;

File: pupil-spa/src/app/login/login.component.html [hunk 2]
```diff
@@ -8,8 +8,8 @@
 
 <div class="grid-row">
     <div class="column-two-thirds">
-        <header class="page-header">
-            <h1 class="heading-xlarge">Multiplication tables check</h1>
+        <header class="page-header" style="margin-top: 25px">
+            <h1 class="heading-xlarge" style="margin-top: 25px">Multiplication tables check</h1>
```
Comment (GeorgeChatzigiannis) [inline pupil-spa/src/app/login/login.component.html]: @jon-shipley I'd rather avoin inline styles.
I don't think they are easily maintainable unless we are using them in common components.

Comment (jon-shipley) [reply pupil-spa/src/app/login/login.component.html]: I usually agree with you.  I could create a new class just for this individual heading, but won't the css size increase? It seems better to acknowledge that this is essentially an override only for this page, and using the style attribute keeps the meaning clear.

Comment (GeorgeChatzigiannis) [reply pupil-spa/src/app/login/login.component.html]: How about that login.component.css?
It seems empty and by default angular creates that for you right?
Personally I find the idea of component based css great.
What are your thoughts on this idea?

Comment (jon-shipley) [reply pupil-spa/src/app/login/login.component.html]: @GeorgeChatzigiannis - obviously its a good idea, but as we aren't using that pattern I don't see that we should start.

Comment (GeorgeChatzigiannis) [reply pupil-spa/src/app/login/login.component.html]: @jon-shipley I don't see us using the inline pattern either though apart from some 
'<br style="clear:both"/>' which are very wrong in the first place.

Comment (pris54) [reply pupil-spa/src/app/login/login.component.html]: @jon-shipley I agree with you that we need to be careful about increasing the css size but inline css is not a good practice. If we write good CSS we would rarely end up with classes that are used only once. If too many of these one times cases are happening then it's time to review our SASS/CSS.

Comment (GuyHarwood) [reply pupil-spa/src/app/login/login.component.html]: @jon-shipley @GeorgeChatzigiannis @pris54 be aware that i am aiming to disallow inline css & JS through content security policy restrictions.  Best we don't add any more going forward.

**Pull request merged**